docs/development/development-environment/macos/README.md

```diff
@@ -172,6 +173,10 @@ It should look like this (just with your database name, username, and password):
 default: &default
   adapter: postgresql
   encoding: unicode
+  # Socket encryption must be disabled on macOS. There is an old bug in which forked processes cause problems on a Mac.
+  # TL;DR: set this flag, otherwise Ruby will crash whenever a route is accessed.
+  # Visit https://github.com/ged/ruby-pg/issues/311 to enter the rabbit hole.
+  gssencmode: disable
```

Comment:
Is this preferred over setting it on your shell profile with `export PGGSSENCMODE="disable"`? If I recall correctly, this is only a problem for Apple Silicon machines.

You might also run into other Apple Silicon issues that can be solved with adding `export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=yes` to your shell profile

Reply:
> Some little postcards from the rabbit hole:
> https://github.com/rails/rails/issues/38560

Is where I first stumbled upon a couple months ago